include/ttmlir/Conversion/Passes.td

```diff
@@ -15,6 +15,12 @@ let summary = "Convert StableHLO dialect to TTIR dialect.";
 }
 #endif
 
+def TTIRGatherPatternMatch : Pass<"convert-ttir-to-ttir", "::mlir::ModuleOp"> {
```

Comment:
This type of classes/transformations are defined in `include/ttmlir/Dialect/TTIR/Transforms/Passes.td`